graphql-server/src/schemas/custom.ts

```diff
@@ -68,15 +135,36 @@ export async function buildCustomSchema(ogm: OGM) {
           sort,
           limit,
           offset,
+          phiEnabled,
         }: QueryDashboardPatientsArgs
       ) {
         const queryBody = buildPatientsQueryBody({ searchVals, columnFilters });
-        return await queryDashboardPatients({
+        const queryFinal = buildPatientsQueryFinal({
           queryBody,
           sort,
           limit,
           offset,
         });
+        const patientsDataPromise = queryDashboardPatients(queryFinal);
+        // Search values are required for PHI searching to restrict PHI data to specific patients
+        // searched for by users
+        if (!phiEnabled || !searchVals || searchVals?.length == 0) {
```

Comment:
I'm seeing this kind of check in multiple places - let's consolidate into a function call?